### ISSUE TITLE
Status of the CFHT calibDB

### ISSUE THREAD

**Issue by pfouque58** [njcuk9999/apero-drs#572]
I have been reprocessing the calibrations used at CFHT for calibrating the various TC and science runs, namely one calibration per run, using v0.5.000 of the DRS. The original list comes from Neil I think and is based on a random selection close to the middle of the run.

At the beginning I have tried to use an empty calibDB so that the results do not depend on which calibration are used to reduce a given calibration. I have also tried to go back in time to avoid Neil's problem with a wrong May 2018 calibration affecting the more recent runs.

Each night generally contains 2 calibration sets: one from the afternoon the other from the next morning. Problems may affect one but not the other, so that a date for which I had problems may in fact be rescued if the correct set is selected only. All the shapes look good to my eye.

Comments on individual calibrations:
Science runs:
19AQ17-Jun 14: FF rms problem: see issue #570 
19AQ10-May14: idem
19AQ08-Apr19: no problem
19AQ02-Feb24: no problem
TC runs:
18BQ24-Jan16: no problem
18BQ21-Dec17: contains in fact 3 calibrations sets
WAVE_B failed in 2362446a_2449c
WAVE_A failed in 2362556a_2559c
2362583a_2586c seems correct
18BQ14-Oct24: no problem
18BQ08-Sep22:
WAVE_AB and WAVE_A fail in 2305305a_5308c
WAVE_B fails in 2305465a_5468c
18AQ18-Jul31:
WAVE_C fails in 2294011a_4914c
WAVE_A and WAVE_C fail in 2295042a_5045c
18AQ10-May26: as far as I understand this is the calibration reported by Neil as causing trouble to the entire calibDB (Neil can you confirm?). I didn't find any problem with it: ???
18AQ07-Apr29: no problem !

It would be useful to investigate more in details each failure: generally the Littrow test is not passed.

It would be useful to suggest other nights of observation to replace the faulty nights, but I have no time to investigate this systematically. A cheap alternative is to remove the faulty calibrations and use successful calibrations from a previous or following run to reduce a given one.

Bottom line is that the calibDB used at CFHT is corrupted for several runs. I don't know how this affects the telluDB based on this calibDB and how this reduces the RV precision we can get for a given star.

Any advice on the best strategy to build a pristine calibDB and then an efficient telluDB will be rewarded.
